src/Hyperion/Hyperion.csproj

```diff
@@ -20,11 +20,7 @@
     <Reference Include="Microsoft.CSharp" />
   </ItemGroup>
 
```

Comment:
#145 
  <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard1.6' ">	  <PropertyGroup Condition=" '$(TargetFramework)' == 'net452' ">
    <DefineConstants>$(DefineConstants);NETSTANDARD</DefineConstants>	
  </PropertyGroup>

Maybe it is because The NETSTANDARD constrant is deleted here

Reply:
Cc @blind-striker

Reply:
The tests should have failed on this case, I'll inform you after check it @DaniilSokolyuk @Aaronontheweb @humhei

Reply:
Looks like the issue is specific to .NET Core 3+

Reply:
Based on what I've found with https://github.com/akkadotnet/Hyperion/pull/151 thus far